src/components/Learn-Components/TOC-Chapters/toc.style.js

```diff
@@ -25,6 +28,23 @@ const TOCWrapper = styled.div`
         }
     }
 
+    .toc-toggle-btn{
+        display:none;
+    }
+     
+    .toc-ul-open{
+        height:auto !important;
+        opacity:1 !important;
+        transition:all .4s !important;
+    }
+
+    .toc-menu-icon{
+        width: 1.5rem; 
+        height: 1.5rem; 
+        cursor: pointer;
+        fill: ${props => props.theme.menuColor};
+    }
```

Comment:
Keep the CSS within their respective parent class's CSS, that way `! important` won't be needed at all.
Always prefer to avoid the usage of `! important` wherever you can.

Reply:
Got it. Thank you. I will make the necessary changes.